src/redux/actions/Dashboard/index.js

```diff
@@ -0,0 +1,38 @@
+import {
+    getDashboardTotals,
+    getDashboardChartsData,
+    getDashboardLastMessages,
+    getDashboardNewUsers,
```

Comment:
resolveTotals, resolveChartsData, etc.
you don't need to specify dashboard since services are inside a dashboard data